lib/beaker/host.rb

```diff
@@ -224,13 +225,29 @@ def is_x86_64?
     end
 
     def connection
-      @connection ||= SshConnection.connect( reachable_name,
+      # create new connection object if necessary
+      @connection ||= SshConnection.connect( { :ip => self['ip'], :vmhostname => self['vmhostname'], :hostname => @name },
```

Comment:
Should this call `ip` method instead of the host param (`self['ip']`)? Or will `self['ip']` trigger the above `ip` method anyway.